src/averageAges.js

```diff
@@ -15,6 +15,18 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
+  const men = people.filter((person) => (person.sex === 'm'));
+  let newpeople;
```

Comment:
Pay attention for variable naming. Use `camelCase` 
```suggestion
  let newPeople;
```